client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -139,11 +180,9 @@ class PublishingTools extends Component {
 				<SettingsSectionHeader title={ translate( 'Publishing Tools' ) } />
 
 				<Card className="publishing-tools__card site-settings__module-settings">
```

Comment:
```suggestion
```
…and remove the `Card` component here, because all the render methods now return `CompactCard`s